azkaban-common/src/test/resources/sample_flow20_01/sample_flow20.yml

```diff
@@ -0,0 +1,41 @@
+---
+# All flow level properties here
+config:
+  flow-level-parameter: value
+
+# This section defines the list of jobs
+# A node can be a job or a flow
+# In this example, all nodes are jobs
+nodes:
```

Comment:
I remember @chengren311 was proposing some condition flow check. Is it easy for him to extend on this yml file?